oaknational/Oak-Web-Application
# fix(curric): update filter spacing

## Description
- Update spacing of child subjects, domain and tier filters in sequence visualiser / unit grid

## How to test

1. Go to [teachers/curriculum/science-secondary/units](https://deploy-preview-1916--oak-web-application.netlify.thenational.academy/teachers/curriculum/science-secondary/units)
2. Scroll down to Year 10 unit section
3. You should see correct spacing for child subject and tier filter buttons
4. You should see correct sizing for tier filter buttons
5. Go to [teachers/curriculum/english-primary/units](https://deploy-preview-1916--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-primary/units)
6. You should correct spacing for domain buttons

## Screenshots

How it used to look (delete if n/a):
<img width="970" alt="Screenshot 2023-09-20 at 11 59 58" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/a3ec9ef6-33c4-40f5-a1fe-ea613946fbd1">
<img width="970" alt="Screenshot 2023-09-20 at 12 01 41" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/25d50708-588e-41ef-ade8-d4c637729b02">

How it should now look:
<img width="970" alt="Screenshot 2023-09-20 at 11 59 37" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/41eafc21-ceab-43b7-b966-52c6cc46e3b0">
<img width="970" alt="Screenshot 2023-09-20 at 12 01 06" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/abcb6716-e42a-4be4-bcdb-6c1a3bf5f064">

## Checklist
